controller/rest/rest.go

```diff
@@ -1677,7 +1677,9 @@ func StartRESTServer() {
 	r.PATCH("/v1/scan/registry/:name", handlerRegistryConfig)
 	r.PATCH("/v2/scan/registry/:name", handlerRegistryConfig)
 	r.POST("/v1/scan/registry/:name/test", handlerRegistryTest)         // debug
+	r.POST("/v2/scan/registry/:name/test", handlerRegistryTest)         // debug
 	r.DELETE("/v1/scan/registry/:name/test", handlerRegistryTestCancel) // debug
+	r.DELETE("/v2/scan/registry/:name/test", handlerRegistryTestCancel) // debug
```

Comment:
DELETE("/v2/scan/registry/:name/test") is not necessary as it's exactly the same as DELETE("/v1/scan/registry/:name/test"